core/src/main/java/pl/agh/edu/utils/ScenariosAdapter.java

```diff
@@ -0,0 +1,23 @@
+package pl.agh.edu.utils;
+
+import pl.agh.edu.enums.DifficultyLevel;
+import pl.agh.edu.enums.HotelType;
+import pl.agh.edu.screen.ScenariosScreen;
+
+public class ScenariosAdapter {
+	private HotelType hotelType;
+	private DifficultyLevel difficultyLevel;
+
+	public void extractDataFromScenariosScreen(ScenariosScreen screen) {
+		hotelType = screen.selectedScenarioButton.get().hotelType;
```

Comment:
nie powinno się robić Optional.get() bez Optional.isPresenet(). Intellij ci tego nie podświetla?

Reply:
To w sumie zamierzone ale poprawię